FreeSurfer/LongitudinalFreeSurferPipeline.sh

```diff
@@ -0,0 +1,418 @@
+
+opts_AddOptional '--seed' 'recon_all_seed' "Seed" 'recon-all seed value'
+
+#TSC: repeatable options aren't currently supported in newopts, do them manually and fake the help info for now
+opts_AddOptional '--extra-reconall-arg-base' 'extra_reconall_args_base' 'token' "(repeatable)  Generic single token argument to pass to recon-all for base template preparation.  Provides a mechanism to:  (i) customize the recon-all command  (ii) specify the recon-all stage(s) to be run (e.g., in the case of FreeSurfer edits)  If you want to avoid running all the stages inherent to the '-all' flag in recon-all,  you also need to include the --existing-subject flag.  The token itself may include dashes and equal signs (although Freesurfer doesn't currently use  equal signs in its argument specification).  e.g., --extra-reconall-arg-base=-3T is the correct syntax for adding the stand-alone '-3T' flag to recon-all.  But, --extra-reconall-arg-base='-norm3diters 3' is NOT acceptable.  For recon-all flags that themselves require an argument, you can handle that by specifying  --extra-reconall-arg-base multiple times (in the proper sequential fashion).  e.g., --extra-reconall-arg-base=-norm3diters --extra-reconall-arg-base=3  will be translated to '-norm3diters 3' when passed to recon-all."
+
+opts_AddOptional '--generate_template_only' 'generate_template_only' 'integer' 'Set to 1 to generate template only without timepoints [0]'
+opts_AddOptional '--generate_timepoints_only' 'generate_timepoints_only' 'integer' 'Set to 1 to generate timepoints only (assuming the template has been generated previously with "--generate_template_only" option) [0]'
+
+opts_AddOptional '--extra-reconall-arg-long' 'extra_reconall_arg_long' 'token' "(repeatable)  Generic single token argument to pass to recon-all for the actual longitudinal processing.  See the description for extra_reconall_arg_base parameter for extra details."
+
+opts_ParseArguments "$@"
+
+if ((pipedirguessed))
+then
+    log_Err_Abort "HCPPIPEDIR is not set, you must first source your edited copy of Examples/Scripts/SetUpHCPPipeline.sh"
+fi
+
+#TSC: hack around the lack of repeatable option support, use a single string for display
+extra_reconall_args_base=${extra_reconall_args_base_manual[*]+"${extra_reconall_args_base_manual[*]}"}
+extra_reconall_args_long=${extra_reconall_args_long_manual[*]+"${extra_reconall_args_long_manual[*]}"}
+
+#display the parsed/default values
+opts_ShowValues
+
+#TSC: now use an array for proper argument handling
+extra_reconall_args_base=(${extra_reconall_args_base_manual[*]+"${extra_reconall_args_base_manual[*]}"})
+extra_reconall_args_long=(${extra_reconall_args_long_manual[*]+"${extra_reconall_args_long_manual[*]}"})
+
+
+echo "extra_reconall_args_base: ${extra_reconall_args_base[@]}"
+echo "extra_reconall_args_long: ${extra_reconall_args_long[@]}"
+
+${HCPPIPEDIR}/show_version
```

Comment:
Redundant with show_tool_versions.

Reply:
Though, it might be better to keep it here and remove it from show_tool_versions instead, in case it errors during configure_custom_tools.

Reply:
ok